### PR TITLE
Modified for changes in recipe

### PR DESCRIPTION
**Pull request by ArchanaShukla**

### CODE REVIEW

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ In this tutorial mission, setup and use these services to automate and achieve o
 
 ---
 
-[ACCORDION-BEGIN [Step 1: ](Check Entitlements)]
+[ACCORDION-BEGIN [Step 1: ](Setup services in your account using Recipe)]
 
-To create service instances for business process management services, you need to ensure that you have the right set of entitlements in your trial account.
+You will use the **recipe** to automatically setup of workflow, business rules, and process visibility services in your account.
```
Comment (MichaelCzcz) [inline tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md:25]: Please rephrase.

- for automatic setup of...
or
- to automatically set up the workflow...

Comment (MichaelCzcz) [inline tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md:25]: You are using `**Recipe**` below. Which version is correct?

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md [hunk 2]
```diff
@@ -38,59 +38,61 @@ To create service instances for business process management services, you need t
...
     >![Account setup](02_Foundation20Onboarding_Processing.png)
 
-3. From your global account page, choose the `trial` tile to access your subaccount.
+3. From your global account page, choose the **Recipe** from left-hand navigation. Among the available recipes, click **Start Recipe** of **Set up account for Workflow Management**.
 
-    !![Enter Trial Subaccount](entertrialaccount.png)
+    !![Start Recipe](startrecipe_2.png)
 
-4. Click **Entitlements** from left panel to check if you have all the below listed entitlements in your subaccount. These entitlements are needed to run other tutorials of this mission.
+4. Recipe will be started with pre-configured steps.
```
Comment (MichaelCzcz) [inline tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md:45]: `**Recipe**` ?

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md [hunk 3]
```diff
@@ -38,59 +38,61 @@ To create service instances for business process management services, you need t
...
-    > If you do not see these entitlements in your subaccount then follow this [tutorial](https://developers.sap.com/tutorials/cp-trial-entitlements.html) that describes how to configure entitlements in your account.
+    - Wait until you see the success popup window once the recipe completes successfully. **Close** the popup.  
 
+    !![Recipe In Progress](startrecipe_3.png)
 
-[DONE]
-[ACCORDION-END]
 
-[ACCORDION-BEGIN [Step 2: ](Download multitarget application file)]
+    > This automatic set up will do the following:
```
Comment (MichaelCzcz) [inline tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md:54]: setup

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md [hunk 4]
```diff
@@ -102,47 +104,33 @@ Download the `BPMServicesFLP.zip` from  [GitHub](https://github.com/SAP-samples/
...
 
-[DONE]
-[ACCORDION-END]
-
-[ACCORDION-BEGIN [Step 5: ](Build and deploy)]
-
-When you build and deploy this application, new service instance will be created for each workflow, business rules and process visibility service in Cloud Foundry.
-
-1. Right-click on the `BPMServicesFLP` project and choose any of the **Build** options.
+9. Right-click on the `BPMServicesFLP` project and choose the **Build** option.
```
Comment (MichaelCzcz) [inline tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md:122]: Don't say right-click on, just say right-click

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md [hunk 5]
```diff
@@ -156,126 +144,35 @@ When you build and deploy this application, new service instance will be created
...
 
-4. **Save** the destination.
-
-
-[DONE]
-[ACCORDION-END]
-
-
-[ACCORDION-BEGIN [Step 8: ](Create destination for Business Rules APIs)]
+3. In **Destination Configuration** section, click on **Edit** and enter your trial password:
```
Comment (MichaelCzcz) [inline tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md:164]: Don't say "click on", just say "click".

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md (hunk 1 above)
Comment (ArchanaShukla) [reply tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md:25]: recipe in above is the term but below is the Button name in the tool.

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md (hunk 2 above)
Comment (ArchanaShukla) [reply tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md:45]: Its Recipe option that appears in the cockpit.

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md (hunk 1 above)
Comment (ArchanaShukla) [reply tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md:25]: Done

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md (hunk 3 above)
Comment (ArchanaShukla) [reply tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md:54]: Done

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md (hunk 4 above)
Comment (ArchanaShukla) [reply tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md:122]: Done

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md (hunk 5 above)
Comment (ArchanaShukla) [reply tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md:164]: Done

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md (hunk 2 above)
Comment (MichaelCzcz) [reply tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md:45]: Sorry, my comment was targeting  the word's formatting.

**Pull request merged**